apstra/test_clients_slicer_test.go

```diff
@@ -0,0 +1,270 @@
+//go:build integration
+// +build integration
+
+package apstra
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+const (
+	slicerTopologyUrlById   = "http://slicer-topology-management-ui.k8s-ci.dc1.apstra.com/v1_1/systest/%s"
+	envSlicerTopologyIdList = "SLICER_TOPOLOGIES"
+)
+
+type slicerDeviceType string
+
+func (o slicerDeviceType) platform() AgentPlatform {
+	switch o {
+	case vSwitchTypeArista:
+		return AgentPlatformEOS
+	case vSwitchTypeNexus:
+		return AgentPlatformNXOS
+	case vSwitchTypeQfx:
+		return AgentPlatformJunos
+	case vSwitchTypeVmx:
+		return AgentPlatformJunos
+	default:
+		return AgentPlatformNull
+	}
+}
```

Comment:
Looks like this method (and the place where it's called) don't make any sense right now.

The eventual implementation of something like this will probably be different enough to warrant killing off this code for now.